server/service/src/sync/test/test_data/stocktake.rs

```diff
@@ -29,7 +29,9 @@ const STOCKTAKE_1: (&str, &str) = (
       "stock_take_time": 47061,
       "store_ID": "store_a",
       "type": "",
-      "om_created_datetime": ""
+      "om_created_datetime": "",
+      "counted_by": "",
+      "verified_by": ""
```

Comment:
i mean i guess no harm, but shouldn't these be `om_` fields?

Reply:
You're right~ Will change